app/utils/totp.server.ts

```diff
@@ -18,7 +18,7 @@
  * Here's the typical process:
  * 1. Generate a secret (crypto.randomBytes(32).toString('hex'))
  * 2. Generate the TOTP with that secret (generateTOTP(secret))
- * 3. Store the secret and the TOTP in the database along with the thing you're verifying (e.g. user email)
+ * 3. Store the secret, digits, period in the database along with the thing you're verifying (e.g. user email)
```

Comment:
I think it should be "Store the secret, digits, **and** period in the database..." 👍

Reply:
```suggestion
 * 3. Store the secret, digits, and period in the database along with the thing you're verifying (e.g. user email)
```